server/src/utils/mixerConnections/AtemConnection.ts

```diff
@@ -0,0 +1,415 @@
+import { Atem, AtemState } from 'atem-connection'
+
+import { store, state } from '../../reducers/store'
+import {
+    fxParamsList,
+    IMixerProtocol,
+} from '../../../../shared/src/constants/MixerProtocolInterface'
+import { logger } from '../logger'
+import { storeSetMixerOnline } from '../../../../shared/src/actions/settingsActions'
+import { IFader } from '../../../../shared/src/reducers/fadersReducer'
+import { IChannel } from '../../../../shared/src/reducers/channelsReducer'
+import { dbToFloat, floatToDB } from './LawoRubyConnection'
+import {
+    AudioMixOption,
+    FairlightAudioMixOption,
+    FairlightInputConfiguration,
+} from 'atem-connection/dist/enums'
+import {
+    storeFaderLevel,
+    storeInputGain,
+    storeSetMute,
+    storeSetPgm,
+    storeSetVo,
+} from '../../../../shared/src/actions/faderActions'
+import { storeSetChLabel } from '../../../../shared/src/actions/channelActions'
+import { FairlightAudioSource } from 'atem-connection/dist/state/fairlight'
+
+export class AtemMixerConnection {
+    private _connection: Atem
+
+    private _chNoToSource: Record<number, number> = {}
+    private _sourceToChNo: Record<number, number> = {}
+    private _sourceTracks: Record<number, '-65280' | '-255' | '-256'> = {}
```

Comment:
Out of curiosity, what does the 3 numbers mean/represent?

Reply:
All stereo inputs can be split into 2 separate channels for individual control

- -65280: control a stereo channel
- -255: control left channel
- -256: control right channel